catalog/dags/common/storage/media.py

```diff
@@ -242,19 +246,19 @@ def _flush_buffer(self) -> int:
         return buffer_length
 
     @staticmethod
-    def _tag_blacklisted(tag: str | dict) -> bool:
+    def _tag_denylisted(tag: str | dict) -> bool:
         """
-        Determine if the is banned or contains a banned substring.
+        Determine if the tag is banned or contains a banned substring.
 
         :param tag: the tag to be verified against the blacklist
         :return: true if tag is blacklisted, else returns false
         """
         if isinstance(tag, dict):  # check if the tag is already enriched
```

Comment:
I realized that removing the check for if the tag is enriched in `_format_raw_tag`:
```
     if isinstance(tag, dict) and tag.get("name") and tag.get("provider"):
            logger.debug(f"Tag already enriched: {tag}")
            return tag
```
means that we also need to remove this check here on lines 256-257.

The tags passed to a `MediaStore` should be lists (or sets) of strings, not lists of dictionaries (enriched tags).

Reply:
Good observation, thanks!